app/api.py

```diff
@@ -345,3 +348,78 @@ async def tech_stack_graph():
     mentees_df["user_role"] = "Mentee"
     df = pd.concat([mentees_df, mentors_df], axis=0).reset_index(drop=True)
     return json.loads(tech_stack_by_role(df).to_json())
+
+
+@API.post("/query/Feedback")
```

Comment:
This endpoint is trying to do too much which leads to it being very complex and hard to use correctly, let's see if we can break it down into 'create', 'read', update' and 'delete' endpoints - this will allow us to use the proper http request methods for each rather than making everything a complicated post request.

Reply:
Can you also take a look at the bottom of this PR where it says this branch has conflicting files between api.py and schema.py? I tried to take a look and I have no idea what it means. Thank you

Reply:
We're also not imposing any data validation on this endpoint's input, and this could lead to very messy data. The standard interchange format for this project is json, all of our endpoints should take and return json. Pydantic provides robust data validation tools, let's rethink this.